src/Expensify.js

```diff
@@ -81,12 +82,8 @@ class Expensify extends PureComponent {
         // Run any Onyx schema migrations and then continue loading the main app
         migrateOnyx()
             .then(() => {
-                // When we don't have an authToken we'll want to show the sign in screen immediately so we'll hide our
-                // boot screen right away
-                if (!this.getAuthToken()) {
-                    this.hideSplash();
```

Comment:
We wait for `react-navigation` to be ready. So there's no need to hide the splash screen here.